test-suite/testkit/src/server.rs

```diff
@@ -149,9 +153,9 @@ impl Handler<RollBack> for TestKitActor {
 
     fn handle(&mut self, RollBack(height): RollBack, _ctx: &mut Self::Context) -> Self::Result {
         if height == Height(0) {
-            return Err(api::Error::BadRequest(
-                "Cannot rollback past genesis block".into(),
-            ));
+            return Err(api::Error::new(api::HttpStatusCode::BAD_REQUEST)
+                .title("RollBack handler failed.")
```

Comment:
Nit: I'd move "Cannot rollback past genesis block" to `title` and leave `detail` empty. IMO, this would make the error easier to understand: the client may not be aware about `RollBack` type, and it's not its handler fault that the request has failed.